src/core/messenger.ts

```diff
@@ -374,9 +372,7 @@ const onMessage = async (event: MessageEvent): Promise<void> => {
 				respond(message.id, event.source, null, response);
 			})
 			.catch((ex: Error) => {
-				reportError(ex, {
-					feature: 'native-ads',
-				});
+				reportError(ex, 'native-ads');
```

Comment:
Note: we are losing the [`shouldThrow`](https://github.com/guardian/commercial/blob/main/src/utils/report-error.ts#L19) flag. Therefore `reportError` will no longer automatically re-throw this error, it won't stop execution and it won't report the error to browser console.